source/base/DateTime.ooc

```diff
@@ -41,6 +41,7 @@ DateTime: cover {
 	_ticks: ULong = 0
 	ticks ::= this _ticks
 	init: func@ (=_ticks)
+	init: func@ ~fromNanoseconds (nanoseconds: ULong) { this _ticks = nanoseconds / This nanosecondsPerTick }
```

Comment:
I mean `ticks` will be whatever we want it to be, but we can rename it to something better so people don't mistake it for .NET ticks.